requirements_stac.txt

```diff
@@ -0,0 +1 @@
+stac-validator>=2.2.0
```

Comment:
Is ist possible to add this to setup.cfg ? Then no requirements file would be needed.

Reply:
It's there, I forgot to delete this file